cli_helpers/tabular_output/terminaltables_adapter.py

```diff
@@ -25,4 +24,10 @@ def adapter(data, headers, table_format=None, **kwargs):
     table = table_format_handler[table_format]
 
     t = table([headers] + list(data), **filter_dict_by_key(kwargs, keys))
-    return t.table
+
+    dimensions = terminaltables.width_and_alignment.max_dimensions(
+        t.table_data,
+        t.padding_left,
+        t.padding_right)[:3]
+    for row in t.gen_table(*dimensions):
+        yield ''.join(''.join(row))
```

Comment:
Is there an extra `join` here? From what I can tell, it looks like a single join is all that's needed.